### PR TITLE
feat(common): translation keys for common components

### PR DESCRIPTION
**Pull request by jalbertsr**
<br/><br/><br/><url>LiveURL: https://smartfaq-common-components-keys-trans.surge.sh</url>

### CODE REVIEW

File: src/common/ContactPageLink.js [hunk 1]
```diff
@@ -19,7 +20,7 @@ const ContactPageLink = ({
   return (
     <div className="contactUs">
       <a target="_blank" href={`/${language}/content/feedback`}>
-        {text}
+        <Trans t={text} />
```
Comment (vborszeki) [inline src/common/ContactPageLink.js]: I would rename this _text_ prop to something like _translationKey_ and then I would pass consistently just translation keys to `<ContactPageLink />` component.
There are some occurrences now, where just simple strings are passed (like in src/SingleBookingPage/bookingItem/Contact.js), which wont't be translated, and the default value for the _text_ prop itself is just a static string too.
What do you think?

Comment (kantord) [reply src/common/ContactPageLink.js]: I agree with @vborszeki

Comment (jalbertsr) [reply src/common/ContactPageLink.js]: Agree, thanks for the suggestion.

Comment (jalbertsr) [reply src/common/ContactPageLink.js]: fixed.

**Pull request merged**